src/app/models/application.go

```diff
@@ -0,0 +1,15 @@
+package models
+
+import r "github.com/dancannon/gorethink"
+
+// UserQuerier queries the user table and logs users in.
+type ApplicationQuerier struct {
```

Comment:
Since we decided to keep just one `Querier` for simplicity, I would revert the commit that separates this from `UserQuerier`.  It might be good to rename `UserQuerier` though.  And we can move the application-specific methods to this file because they're all in the same package.